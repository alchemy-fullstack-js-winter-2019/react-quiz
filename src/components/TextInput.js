import React from 'react';
import PropTypes from 'prop-types';

function TextInput({ handleChange }) {
  return (
    <>
        <input type="text" name="text" onChange={handleChange}></input>
    </>
  );
}

TextInput.propTypes = {
  handleChange: PropTypes.func.isRequired  
};

export default TextInput;
