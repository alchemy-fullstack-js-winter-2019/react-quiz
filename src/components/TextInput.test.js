import React from 'react';
import renderer from 'react-test-renderer';
import TextInput from './TextInput';

describe('TextInput', () => {
  it('matches a snapshot', () => {
    const tree = renderer.create(
      <TextInput />
    ).toJSON();

    expect(tree).toMatchSnapshot();
  });
});


