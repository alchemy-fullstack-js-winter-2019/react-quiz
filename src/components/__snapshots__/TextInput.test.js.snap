// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TextInput matches a snapshot 1`] = `
<input
  name="text"
  type="text"
/>
`;
